### PR TITLE
fix(eval): add leftcol in getwininfo

### PR DESCRIPTION
**Pull request by glepnir**
Problem: currently no way get leftcol of a special window

Solution: add leftcol in getwininfo eval function

### CODE REVIEW

File: src/testdir/test_bufwintabinfo.vim [hunk 1]
```diff
@@ -115,6 +115,18 @@ func Test_getbufwintabinfo()
...
+function Test_get_wininfo_leftcol()
+  set nowrap
+  set winwidth=10
+  vsp
+  call setline(1, ['abcdefghijklmnopqrstuvwxyz'])
+  norm! 5zl
+  call assert_equal(5, getwininfo()[0].leftcol)
+  set wrap&
+  set winwidth&
+  close
```
Comment (zeertzjq) [inline src/testdir/test_bufwintabinfo.vim]:
```suggestion
  bwipe!
  set wrap&
  set winwidth&
```

**Comment by zeertzjq** [comment]
This is not documented

**Comment by glepnir** [comment]
winsaveview does not take a window id as param

File: runtime/doc/builtin.txt [hunk 2]
```diff
@@ -5094,6 +5094,8 @@ getwininfo([{winid}])					*getwininfo()*
 					{only with the +terminal feature}
 			tabnr		tab page number
 			topline		first displayed buffer line
+			leftcol		first column displayed; only used when
+					'wrap' is off
```
Comment (zeertzjq) [inline runtime/doc/builtin.txt]: This list is sorted alphabetically

**Comment by chrisbra** [comment]
Yes true. Are there other keys from `winsaveview()` that we need?

File: runtime/doc/builtin.txt (hunk 2 above)
Comment (glepnir) [reply runtime/doc/builtin.txt]: Aha got it.

**Comment by chrisbra** [comment]
alright thank

**Pull request closed without merging**